corehq/apps/users/views/mobile/users.py

```diff
@@ -743,6 +748,9 @@ def post(self, request, *args, **kwargs):
             if 'location_id' in request.GET:
                 user.set_location(loc)
 
+            if phone_number:
+                initiate_sms_verification_workflow(user, phone_number)
```

Comment:
Also, I think you'll only want to do this if the project has access to inbound and outbound sms